student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(name = 'Unknown', age = nil, classroom:, parent_permission: true)
```

Comment:
- [ ] I still got the same error that the previous reviewer detected, kindly ensure that the **`classroom`** variable has a value attached to it or you remove the **`:`** notation. ℹ️ 

<img width="473" alt="Untitled" src="https://github.com/ClintonYounge/School-Libraby-Project/assets/103386183/0360872a-7f3d-4764-825b-a936d12423c9">

```suggestion
  def initialize(name = 'Unknown', age = nil, classroom, parent_permission: true)
```